src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -266,4 +299,11 @@ const StyledMenu = styled(({isOpen, blendCorner, alignMenu, ...props}) => (
   ${({alignMenu}) => (alignMenu === 'left' ? 'left: 0;' : '')};
 `;
 
+const EmptyMessage = styled('div')`
+  opacity: 0.4;
```

Comment:
i would prefer to just change the text color, rather than achieving a text color through opacity.